acceptance/features/migration.feature

```diff
@@ -0,0 +1,25 @@
+Feature: operator can migrate/adopt Redpanda helm base deployment
+
+  @skip:gke @skip:aks @skip:eks
+  Scenario: Adopt or migrate from stand alone helm chart release to Redpanda custom resource
+        Given there is "test-migrate-from-helm" Redpanda helm release with values:
```

Comment:
```suggestion
        Given a helm release "test-migrate-from-helm" of "redpanda/redpanda" with values:
```

We can cheat and map `redpanda/redpanda` to `../charts/redpanda` in the implementation :P

Reply:
I'm not sure if that contradict with 

> I think we'll also see a massive amount of benefit if we specify the version of the chart in this test.